rinja_derive/src/generator/expr.rs

```diff
@@ -1065,18 +1049,50 @@ impl<'a> Generator<'a, '_> {
     }
 }
 
+#[inline]
+fn get_filter_argument<'a, 'b>(
+    ctx: &Context<'_>,
+    name: &str,
+    args: &'b [WithSpan<'b, Expr<'a>>],
+    node: Span<'_>,
+) -> Result<&'b WithSpan<'b, Expr<'a>>, CompileError> {
+    match args {
+        [arg] => Ok(arg),
+        _ => Err(unexpected_filter_arguments(ctx, name, args, node)),
+    }
+}
+
+#[cold]
+#[inline(never)]
+fn unexpected_filter_arguments(
+    ctx: &Context<'_>,
+    name: &str,
+    args: &[WithSpan<'_, Expr<'_>>],
+    node: Span<'_>,
+) -> CompileError {
+    ctx.generate_error(
+        format_args!(
+            "unexpected argument{} in `{name}` filter",
+            if args.len() == 2 { "" } else { "s" },
```

Comment:
```suggestion
            "expected 1 argument in `{name}` filter, found {}",
            args.len(),
```

Better than handling plural: not handling plural. :D

Reply:
Good idea, but the "1" is not true in all cases. :) I reworded the message, and implemented it with less magic. <del>🪄 </del>